fludashboard/libs/views.py

```diff
@@ -169,7 +169,7 @@ def data__weekly_incidence_curve(
         pass
 
     try:
-        min_week = int(df.loc[df['situation_id'] == 0, 'epiweek'].min())
+        min_week = int(df.loc[df['situation_id'] == 4, 'epiweek'].min())
```

Comment:
@xmnlab What is the intent of this comparison?
From the following lines in the code, it appears to be checking where are the incomplete data range. If so, the situation_id is 4, not 0.
Either way, it doesn't seem to change anything...